# CubeMarsFeedback.msg
Header header
float32 position
float32 speed
float32 current
int8 temperature
uint8 error_code
